pkg/applier/manager.go

```diff
@@ -44,6 +45,8 @@ type Manager struct {
 	LeaderElector controller.LeaderElector
 }
 
+var _ component.Component = (*Manager)(nil)
```

Comment:
Almost everywhere we do it like `var _ component.Component = &Manager{}`. I'd prefer to keep it consistent.

Reply:
I'm ok with both, as I said we just need to be consistent here.

Reply:
Opened #1689 for that.